bentoml/lightgbm.py

```diff
@@ -193,9 +204,10 @@ def num_replica(self) -> int:
 
     # pylint: disable=arguments-differ,attribute-defined-outside-init
     def _setup(self) -> None:
-        self._model = lgb.Booster(
-            params=self._booster_params, model_file=self._model_file
-        )
+        if os.path.splitext(self._model_file)[1] == PKL_EXT:
+            self._model = joblib.load(self._model_file)
```

Comment:
Since now we are also trying to support the SKlearn API, maybe have the `infer_api_callback` to allow user to use `predict_proba` if specified. Otherwise we will just use the default `predict` function. 
```python
def __init__(self, infer_api_callback: str, ...):
	self._infer_api_callback = infer_api_callback
def _setup(self):
	self._predict_fn = getattr(self._model, self._infer_api_callback)

def load_runner(..., infer_api_callback: str = "predict",...):
	return _LightGBMRunner(..., infer_api_callback=infer_api_callback,...)
```